app/models/concerns/processing_labeler.rb

```diff
@@ -90,7 +90,6 @@ def in_transit?(status)
   # expect. This string is based on the item status in Aleph, but we edit it
   # for readability.
   def processing_label(status, z30_item_process, reserve)
-    return 'On reserve (see desk)' if reserve
```

Comment:
Are we using the reserve parameter anywhere else in this method (or related code)? Or can we remove it from the method and the calling code entirely?

Reply:
Good question.

Yes we still use it for some of the general button logic (some of which is deactivated as this time, but will come back one day with some ENV changes when needed).

Reply:
Oh wait, I think I misunderstood your question. We indeed don't use it in this method and can likely clean this up. Let me check (my response was based on we still care about reserves elsewhere but you clearly asked about this method and I answered wrongly... sorry). Great catch! Will check and likely clean this up.

Reply:
@matt-bernhardt I've made the suggested change. Thanks.

Reply:
Yep - that's what I was thinking of. Thanks! Thumbing next